redis_rate_limit/__init__.py

```diff
@@ -37,29 +38,82 @@ class TooManyRequests(Exception):
     pass
 
 
-class RateLimit(object):
+class RateLimiter(object):
```

Comment:
My suggestion here is to make both names work and if old one is used issue a deprecation warning, let's change the class name on a new major version.

Reply:
Ok, I will fix that.